libraries/AP_EFI/AP_EFI_Serial_Hirth.h

```diff
@@ -31,6 +31,28 @@ class AP_EFI_Serial_Hirth: public AP_EFI_Backend {
 
     void update() override;
 
+    enum class Error_Excess_Temp_Bitfield : uint16_t {
```

Comment:
These ones that are only really used for log documentation you can define right in the `log_status` function in the cpp. That stops it getting used elsewhere my mistake.

Reply:
@IamPete1 what i mention above is that ok?

Reply:
seems reasonable to me